<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml"><head><title>R: Data from the Amsterdam Cohort Studies on HIV infection and...</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body>

<table width="100%" summary="page for aidssi"><tr><td>aidssi</td><td style="text-align: right;">R Documentation</td></tr></table>

<h2>Data from the Amsterdam Cohort Studies on HIV infection and AIDS</h2>

<h3>Description</h3>

<p>These data sets give the times (in years) from HIV infection to AIDS, SI
switch and death in 329 men who have sex with men (MSM). Data are from the
period until combination anti-retroviral therapy became available (1996).
For more background information on the cohort, ccr5 and SI, see Geskus
<em>et al.</em> (2000, 2003)
</p>


<h3>Format</h3>

<p>aidssi </p>

<table summary="Rd table">
<tr>
 <td style="text-align: left;"> patnr:</td><td style="text-align: left;"> Patient identification number</td>
</tr>
<tr>
 <td style="text-align: left;">
time:</td><td style="text-align: left;"> Time from HIV infection to first of SI appearance and AIDS, or
last follow-up</td>
</tr>
<tr>
 <td style="text-align: left;"> status:</td><td style="text-align: left;"> Event indicator; 0 = censored, 1 = AIDS, 2 =
SI appearance</td>
</tr>
<tr>
 <td style="text-align: left;"> cause:</td><td style="text-align: left;"> Failure cause; factor with levels "event-free",
"AIDS", "SI"</td>
</tr>
<tr>
 <td style="text-align: left;"> ccr5:</td><td style="text-align: left;"> CCR5 genotype; factor with levels "WW" (wild type
allele on both chromosomes),</td>
</tr>
<tr>
 <td style="text-align: left;"> </td><td style="text-align: left;"> "WM" (mutant allele on one
chromosome)</td>
</tr>
<tr>
 <td style="text-align: left;"> </td>
</tr>

</table>
<p> aidssi2 </p>

<table summary="Rd table">
<tr>
 <td style="text-align: left;"> patnr:</td><td style="text-align: left;"> Patient identification
number</td>
</tr>
<tr>
 <td style="text-align: left;"> entry.time:</td><td style="text-align: left;"> Time from HIV infection to cohort entry. Value is
zero if HIV infection occurred while in follow-up.</td>
</tr>
<tr>
 <td style="text-align: left;"> aids.time:</td><td style="text-align: left;"> Time
from HIV infection to AIDS, or last follow-up if AIDS was not observed</td>
</tr>
<tr>
 <td style="text-align: left;">
aids.stat:</td><td style="text-align: left;"> Event indicator with respect to AIDS, with values 0
(censored) and 1 (AIDS)</td>
</tr>
<tr>
 <td style="text-align: left;"> si.time:</td><td style="text-align: left;"> Time from HIV infection to SI
switch, or last follow-up if SI switch was not observed</td>
</tr>
<tr>
 <td style="text-align: left;"> si.stat:</td><td style="text-align: left;">
Event indicator with respect to SI switch, with values 0 (no switch) and 1
(switch)</td>
</tr>
<tr>
 <td style="text-align: left;"> death.time:</td><td style="text-align: left;"> Time from HIV infection to death, or last
follow-up if death was not observed</td>
</tr>
<tr>
 <td style="text-align: left;"> death.stat:</td><td style="text-align: left;"> Event indicator with
respect to death; 0 = alive, 1 = dead</td>
</tr>
<tr>
 <td style="text-align: left;"> age.inf:</td><td style="text-align: left;"> Age at HIV
infection</td>
</tr>
<tr>
 <td style="text-align: left;"> ccr5:</td><td style="text-align: left;"> CCR5 genotype; factor with levels "WW" (wild type
allele on both chromosomes),</td>
</tr>
<tr>
 <td style="text-align: left;"> </td><td style="text-align: left;"> "WM" (mutant allele on one
chromosome)</td>
</tr>
<tr>
 <td style="text-align: left;"> </td>
</tr>

</table>



<h3>Details</h3>

<p><code>aidssi</code> contains follow-up data until the first of AIDS and SI switch.
It was used as example for the competing risks analyses in Putter, Fiocco,
Geskus (2007) and in Geskus (2016).
</p>
<p><code>aidssi2</code> extends the <code>aidssi</code> data set in three ways. First, it
considers events after the initial one. Second, it includes the entry times
of the individuals that entered the study after HIV infection. Third, age at
HIV infection has been added as extra covariable. Numbers differ slightly
from the <code>aidssi</code> data set. Some individuals were diagnosed with AIDS
only when they died and others had their last follow-up at AIDS diagnosis.
In order to prevent two transitions to happen at the same time, their time
to AIDS was shortened by 0.25 years. This data set was used as example for
the multi-state analyses in Geskus (2016).
</p>


<h3>Source</h3>

<p>Geskus RB (2000). On the inclusion of prevalent cases in HIV/AIDS
natural history studies through a marker-based estimate of time since
seroconversion.  <em>Statistics in Medicine</em> <b>19</b>, 1753&ndash;1769.
</p>
<p>Geskus RB, Miedema FA, Goudsmit J, Reiss P, Schuitemaker H, Coutinho RA
(2003).  Prediction of residual time to AIDS and death based on markers and
cofactors.  <em>Journal of AIDS</em> <b>32</b>, 514&ndash;521.
</p>


<h3>References</h3>

<p>Geskus, Ronald B. (2016). <em>Data Analysis with Competing
Risks and Intermediate States.</em> CRC Press, Boca Raton.
</p>
<p>Putter H, Fiocco M, Geskus RB (2007). Tutorial in biostatistics: Competing
risks and multi-state models. <em>Statistics in Medicine</em> <b>26</b>,
2389&ndash;2430.
</p>


</body></html>
