<!DOCTYPE html><html><head><title>R: P-values from biological expression array data</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.15.3/dist/katex.min.css">
<script type="text/javascript">
const macros = { "\\R": "\\textsf{R}", "\\code": "\\texttt"};
function processMathHTML() {
    var l = document.getElementsByClassName('reqn');
    for (let e of l) { katex.render(e.textContent, e, { throwOnError: false, macros }); }
    return;
}</script>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.15.3/dist/katex.min.js"
    onload="processMathHTML();"></script>
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body><div class="container"><main>

<table style="width: 100%;"><tr><td>coralPval</td><td style="text-align: right;">R Documentation</td></tr></table>

<h2>
P-values from biological expression array data
</h2>

<h3>Description</h3>

<p>P-values were calculated for each of 3072 genes, for data that
compared expression values between post-settlement coral larvae
and pre-settlement coral larvae.
</p>


<h3>Usage</h3>

<pre><code class='language-R'>data("coralPval")</code></pre>


<h3>Format</h3>

<p>The format is:
num [1:3072, 1] 8.60e-01 3.35e-08 3.96e-01 2.79e-01 6.36e-01 ...
</p>


<h3>Details</h3>

<p>t-statistics, and hence p-values, were derived from five replicate
two-colour micro-array slides.  Details are in a vignette that
accompanies the <span class="pkg">DAAGbio</span> package.
</p>


<h3>Source</h3>

<p>See the <code>?DAAGbio::coralRG</code>
</p>


<h3>References</h3>

<p>Grasso, L. C.; Maindonald, J.; Rudd, S.; Hayward, D. C.; Saint, R.;
Miller, D. J.; and Ball, E. E., 2008. Microarray analysis identifies
candidate genes for key roles in coral development. BMC Genomics, 9:540.
</p>


<h3>Examples</h3>

<pre><code class='language-R'>## From p-values, calculate Benjamini-Hochberg false discrimination rates
fdr &lt;- p.adjust(DAAG::coralPval, method='BH')
## Number of genes identified as differentially expressed for FDR = 0.01
sum(fdr&lt;=0.01)
</code></pre>

</main>

</div>
</body></html>
